code/modules/reagents/Chemistry-Reagents/Medicine-Reagents.dm

```diff
@@ -346,6 +346,7 @@
 	for(var/datum/reagent/R in M.reagents.reagent_list)
 		if(R != src)
 			M.reagents.remove_reagent(R.id,2.5)
+			M.client.dir = NORTH
```

Comment:
Instead of explicitly returning the client dir to normal each life just make it purge the riotatium as usual and make the dir return only on removed() of rotatium(or how is this proc called when chem lefts the body i forgot)